content/200-concepts/100-components/01-prisma-schema/04-data-model.mdx

```diff
@@ -891,20 +844,12 @@ model Post {
 }
 ```
 
-#### Index names
```

Comment:
Should this keep the headline to be linkable? 
Should we have a similar headline in the other mentions of customizing the constraint names?